integration-tests/oidc-tenancy/src/main/java/io/quarkus/it/keycloak/CustomTenantConfigResolver.java

```diff
@@ -73,11 +74,24 @@ public OidcTenantConfig get() {
                     // try the absolute URI
                     config.setIntrospectionPath(authServerUri + "/introspect");
                     return config;
+                } else if ("tenant-introspection-multiple-required-claims".equals(tenantId)) {
+                    String uri = context.request().absoluteURI();
+                    String authServerUri = uri.replace("/tenant-introspection/tenant-introspection-multiple-required-claims",
+                            "/oidc");
+                    return OidcTenantConfig
+                            .authServerUrl(authServerUri)
+                            .tenantId("tenant-introspection-multiple-required-claims")
+                            .discoveryEnabled(false)
+                            .clientId("client")
+                            .introspectionPath(authServerUri + "/introspect")
+                            .allowTokenIntrospectionCache(false)
+                            .token().requiredClaims("required_claim", Set.of("1", "2")).end()
```

Comment:
@michalvavrik Michal, I think, to avoid breaking some custom tenant config resolver code (we may have some users setting an old map), let's keep the old method as well but make it deprecated and pass this map to a new method, converting String values to Set, checking for `,` - it is a bit of a hassle, but without it we would need to mark this update as breaking

Reply:
Hmm, that's not going to overload cleanly. May be we need to add a method like `setRequiredArrayClaims` accepting a new Map...

Reply:
Ok.

Reply:
I have added the `setRequiredClaimValues` method so there are no breaking changes on the builder.